src/utils/claimgraph.js

```diff
@@ -23,14 +35,23 @@ export function fromJsonldjsCg(jcg) {
 
 // convert a single node from json-ld claimgraph representation to this module's representation.
 function fromJsonldjsNode(jn) {
+  let ret;
```

Comment:
`ret` declaration can go below as its not used until `case 'Literal'`

Reply:
I did that originally but eslint got upset. It doesn't like block-scoped variables to be declared inside switch statements.